source/Hardpoint.h

```diff
@@ -89,12 +99,16 @@ class Hardpoint {
 	const Outfit *outfit = nullptr;
 	// Hardpoint location, in world coordinates relative to the ship's center.
 	Point point;
-	// Angle of firing direction (guns only).
+	// Angle of firing direction (guns) or holding position (turret).
 	Angle baseAngle;
+	// Range of the capable angle to point to the target (turret only).
+	std::pair<Angle, Angle> movableAngle;
```

Comment:
I think a name like `turnRange` will reflect the meaning of this variable better than movableAngle.

Similar for the comment:
`// Range over which the turret can turn, from leftmost position to rightmost position (turret only)`

Reply:
I changed the name in local before reading your review. It's "angleOfTraverse" now.

Do you think which is name better?

Reply:
Both turnRange as well as angleOfTraverse sound better to me than movableAngle. I'm not a native English speaker, so I leave it to other reviewers to comment on the most proper name.